### DISCUSSION TITLE
Automatic Swear Word Detection & Alerting

### DISCUSSION THREAD

**Discussion by sam-ulrich1** [openai/whisper#297] [Show and tell]
My girlfriend wants me to stop swearing in front of her family (fair). The problem is that I just don't notice it anymore! I built a bot using Whisper and derived from whisper_mic to automatically detect swear words and beep at me.

https://github.com/sam-ulrich1/i-swear-ill-stop

**Comment by Abdurrafey-Siddiqui** [comment]
hey could you make something like where we can add words to the whisper dictionary or whatever it is so that it can recognise. Since like for example in some audio, a person might say a word in a different language but I want whisper to write it as a transliterated version in English. But whisper would most likely transcribe it incorrectly.

**Reply by sam-ulrich1** [reply]
Can you open up a Github issue on the project and we can talk through it?
